packages/fast-components-styles-msft/README.md

```diff
@@ -24,3 +24,89 @@ class Button extends React.Component {
 
 export default manageJss(ButtonStyles)(Button);
 ```
+
+## Color
+
+Color palettes are the core of the how color is applied in MSFT stylesheets. There are two core color palettes that are exposed by the design system that are used to derive color; the *neutral* palette and the *accent* palette. The provided color utilities always derive the colors they return from one of these two palettes. They generally also derive the color from the input *design system*, allowing them to work across a wide variety for background colors.

```

Comment:
End of this sentence reads a little weird to me. Did you mean "for"?

Reply:
sure didn't!